indigo_social/models.py

```diff
@@ -78,3 +78,36 @@ def retrieve_profile_photo_on_signup(sender, **kwargs):
 
     except (SocialAccount.DoesNotExist, UserProfile.DoesNotExist):
         pass
+
+
+class BadgeAward(models.Model):
+    user = models.ForeignKey(User, related_name="badges_earned", on_delete=models.CASCADE)
+    awarded_at = models.DateTimeField(auto_now_add=True)
+    slug = models.CharField(max_length=255)
+
+    class Meta:
+        db_table = "pinax_badges_badgeaward"
```

Comment:
would this also fall away at some point? or is it (and the `options` above) safe to leave in forever?

Reply:
we can leave this as is; the only argument for moving the table is to have a table with a different name